### DISCUSSION TITLE
VIDEO

### DISCUSSION THREAD

**Discussion by xslaayer29** [anidl/multi-downloader-nx#98] [Q&A] [answered]
Hello, I would like to know if there is a video where you explain the installation and how to run the programs

**Comment by izu-co** [comment, marked as answer]
No, there is not. I have started with a wiki but didn't had much time in the past few weeks to continue it.

**Reply by xslaayer29** [reply]
Do not worry, I already achieved it, but now I have a problem, the video and audio come out separately, and also the format is neither mkv nor mp4 so I don't know what to do to modify it exactly

**Reply by izu-co** [reply]
You need to have either ffmpeg or mkvmerge installed (in the correct dir) or you have to merge it manually
